src/components/conditional.js

```diff
@@ -45,9 +45,17 @@
           setVisible(initialVisibility);
         }, [checkCondition]);
 
+        useEffect(() => {
+          if (visible) {
+            B.triggerEvent('isTrue', true);
+          } else {
+            B.triggerEvent('isFalse', false);
+          }
+        }, [visible]);
```

Comment:
This is very nice to be able to build an if else with multiple conditionals, kudos!